test/upsert_integration_test.go

```diff
@@ -87,11 +87,11 @@ func TestUpsertIntegration(t *testing.T) {
 		t.FailNow()
 	}
 
-	test_util.AssertEqual(t, len(response1.Hits.Hits), 1)
-	test_util.AssertEqual(t, response1.Hits.Hits[0].Source.Email, "mario@rossi.it")
-	test_util.AssertNotNull(t, response1.Hits.Hits[0].Source.LastUpdate)
-	test_util.AssertNotNull(t, response1.Hits.Hits[0].Source.UserID)
-	test_util.AssertEqual(t, response1.Hits.Hits[0].ID, "mario@rossi.it")
+	assert.Equal(t, len(response1.Hits.Hits), 1, "Test should extract exactly ONE result from Elastic")
+	assert.Equal(t, response1.Hits.Hits[0].Source.Email, "mario@rossi.it", "Email not valid")
+	assert.NotNil(t, response1.Hits.Hits[0].Source.LastUpdate, "Last Update should not be NIL")
```

Comment:
this is not already done in https://github.com/Ringloop/mr-plow/pull/36/ ?

Reply:
Yes, I'm going to make all of this using "require"